app/src/main/java/org/metabrainz/android/presentation/features/adapters/BlogAdapter.kt

```diff
@@ -22,7 +22,16 @@ class BlogAdapter(private val context: Context, private val posts: ArrayList<Pos
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
         val post = posts[position]
-        holder.heading.text = post.title
+        holder.heading.text = when{
```

Comment:
This does not solve the problem. I checked after running this on a device. Also, please attach the link to the ticket in the descriptions from next time.

Reply:
Thank you very much Sir for considering my code and taking a look on it.
I knew sir there may be problem in some cases.
But I have no choice, until any problem is found. Thanks again for your response.

Reply:
> This does not solve the problem. I checked after running this on a device. Also, please attach the link to the ticket in the descriptions from next time.

Okay Sir, I again create the PR with the issue link.  And I am quite confident with my code, so can you please share me the snapshot of that particular section, where it is not fixed yet,If possible.